src/lang/lwc_tw.properties

```diff
@@ -22,9 +18,6 @@ yes=%dark_green%是
 no=%dark_red%否
 
 # Some generic responses and block names
-## TRANSLATORS: 相關翻譯請參閱 https://crowdin.com/project/minecraft/zh_TW
```

Comment:
Why are you deleting translator comments? These may be relevant to people who are trying to update the translations.